internal/function/is_remote.go

```diff
@@ -18,11 +18,9 @@ func NewIsRemote(e sql.Expression) sql.Expression {
 	return &IsRemote{expression.UnaryExpression{Child: e}}
 }
 
-var _ sql.Expression = (*IsRemote)(nil)
```

Comment:
Why now isn't needed to check that it implements the `sql.Expression` interface?

Reply:
`TransformUp` already makes this check